metrics.go

```diff
@@ -14,9 +14,8 @@ import (
 func (c *collector) clusterMetrics(ch chan<- prometheus.Metric, client *kazoo.Kazoo) {
 	controller, err := client.Controller()
 	if err != nil {
-		msg := fmt.Sprintf("Error collecting cluster controller broker ID: %s", err)
-		log.Error(msg)
-		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("zookeeper_controller_id_error", msg, nil, nil), err)
+		log.Errorf("Error collecting cluster controller broker ID: %s", err)
+		atomic.StoreInt32(&c.zkErr, 1)
```

Comment:
Doesn't make sense to do here because we couldn't retrieve that from zookeeper.

I'm considering putting the zookeeper in a label for the scrape error as part of the future work to configure allowed zookeepers on the command line, but nothing in this exporter is talking to the brokers directly.